docs/intro/quickstart.rst

```diff
@@ -18,34 +18,100 @@ data, and writing your first query. Let's jump in!
 First let's install the EdgeDB CLI. Open a terminal and run the appropriate
 command below.
 
-**macOS/Linux**
+.. note:: Great news for Node users!
 
-.. code-block:: bash
+    Skip installing and start using the EdgeDB CLI right away! Just use ``npx
+    edgedb``. For example, to create a new project, you can use ``npx edgedb
+    project init``. Try it with :ref:`any of our CLI commands
+    <ref_cli_overview>`!
```

Comment:
I could address it here the way I did in the video: "… `npx edgedb` or your package manager's equivalent." That doesn't tell anyone _how_ to do it with another package manager, but it at least tells them it can be done in case they weren't aware.

Reply:
I've revised it to incorporate that for now, and we can do something more robust later if we want.